internal/ingress/metric/collectors/socket.go

```diff
@@ -98,9 +100,11 @@ var (
 // updating the library to latest version changed the output of the metrics
 var defObjectives = map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001}
 
+//var defBuckets = []float64{0.01, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.5, 2.0, 10.0}
```

Comment:
I think we can remove this? If we no longer use it.

Reply:
Fixed too